src/shared/directives/conditional-show/domain/conditional-show.model.ts

```diff
@@ -4,21 +4,34 @@ import { FieldsUtils } from '../../../services/fields';
 export class ShowCondition {
 
   private static readonly AND_CONDITION_REGEXP = new RegExp('\\sAND\\s(?![^"]*"(?:(?:[^"]*"){2})*[^"]*$)', 'g');
+  private static readonly OR_CONDITION_REGEXP = new RegExp('\\sOR\\s(?![^"]*"(?:(?:[^"]*"){2})*[^"]*$)', 'g');
+  private static CONDITION_NOT_EQUALS = '!=';
+  private static CONDITION_EQUALS = '=';
   private static readonly CONTAINS = 'CONTAINS';
 
   static addPathPrefixToCondition(showCondition: string, pathPrefix): string {
     if (!pathPrefix || pathPrefix === '') {
       return showCondition;
     }
-    let andConditions = showCondition.split(ShowCondition.AND_CONDITION_REGEXP);
-    andConditions = andConditions.map(condition => {
+    if (showCondition.search(ShowCondition.OR_CONDITION_REGEXP) !== -1) {
```

Comment:
What will happen when we will mix OR and AND condition? It will only add prefix to OR? Please update test with this scenario.

Reply:
There is no business of mixing OR and AND operations in the same condition. Steve confirmed.